src/screens/IndexScreen.js

```diff
@@ -445,6 +451,25 @@ export const IndexScreen = ({ navigation, route }) => {
                       />
                     )
                   }
+                  ListFooterComponent={
+                    <>
+                      {query === QUERY_TYPES.CATEGORIES && !!categoryListFooter && (
+                        <>
+                          <Wrapper>
+                            <RegularText small>{categoryListFooter.footerText}</RegularText>
+                          </Wrapper>
+                          {categoryListFooter.url && categoryListFooter.buttonTitle && (
```

Comment:
same as in #805, add `!!` for the checks here please to be safe here for all platforms

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/851d3d824c9ca2a82fdc4f5e3ba26050d8114b56